### ISSUE TITLE
Update to R 3.5.0

### ISSUE THREAD

**Issue by dannyparsons** [africanmathsinitiative/R-Instat#4577]
A new major version of R has been released which R-Instat currently doesn't support. We should update to support this new version. It means we will need the "3.5" versions of the R packages for users to install. And this will probably replace the "3.4" versions (otherwise we need two copies of the packages and significantly increase the installer size). However this will mean users won't be able to install packages offline if they have R 3.4.x.

I am assuming we do this for the next version released? If not, then we won't be able to install packages offline for R 3.5.0.

**Comment by maxwellfundi** [comment]
@dannyparsons  Considering we have about a week to the next release, I suggest we start testing this new R Version to see if it works well because having both the 3.4 and 3.5 packages, we will have the size increase to almost double. what @africanmathsinitiative/developers think about this?

**Comment by dannyparsons** [comment]
I agree there should be good testing of R 3.5.0 before updating. Given that next week we have two important workshops there isn't much time to test this properly and so updating before then will be a risk. I suggest we stick with the current 3.4.x versions for this release and plan an update, possible R-Instat 0.5 for a few weeks time once we have tested this out.

**Comment by dannyparsons** [comment]
No the latest version of RDotNet still doesn't include the updates for R 3.5.

**Comment by maxwellfundi** [comment]
Okay, I really hoped this would work. But lets wait and see when it is
updated to handle this version.

Regards
Maxwell Fundi
|| Phone :+254728508241||

**Comment by dannyparsons** [comment]
Their is still discussion on the topic and I'm hoping an update will be released soon.

**Comment by maxwellfundi** [comment]
@dannyparsons  was there any outcome on discussions on this? Are we likely to go to R 3.5.0 soon?

**Comment by dannyparsons** [comment]
Finally, R.NET has been updated to include support for R 3.5.x with the release of an update on nuget https://www.nuget.org/packages/R.NET/1.8.1. I had done some limited testing of the beta release before this and it seemed to work fine in R-Instat. I will make a pull request to update the package so we have time to test thoroughly before our next release.

This also means we can finally move from 0.4.x releases! 
Of course, to complicate things R 3.6.0 has also just been released! So R.NET is untested on this but there wasn't the major change in structure as there was to R 3.5.0 so R.NET should also work with R 3.6.0 but we need to test this very well and decide if we support it before the next release.

We also need to decide on the version number for our next release. We had suggested 0.5.0, to be in line with supporting R 3.5.x, however now there's also R 3.6.0. I think we can still go with 0.5.0 as showing that we support R 3.5.0 and above or we could skip to 0.6.0 if we wanted.
